keras/backend/cntk_backend.py

```diff
@@ -58,7 +62,9 @@ def get_uid(prefix=''):
 
 def learning_phase():
     # If _LEARNING_PHASE is not 0 or 1, return dynamic learning phase tensor
-    return _LEARNING_PHASE if _LEARNING_PHASE in {0, 1} else _LEARNING_PHASE_PLACEHOLDER
+    return (
```

Comment:
Please make multiple statements instead of breaking the line, it will improve readability.
```python
if _LEARNING_PHASE ....:
     return ....
else:
     return ....
```